### DISCUSSION TITLE
context.result.body only returns last value

### DISCUSSION THREAD

**Discussion by PPetterssonSO** [SuperOffice/typescript-for-superoffice#19] [General]
How can we do multiple  context.result.body (printLines)
For example, looping an array and printing out the content, how can this be achived when context.result.body only returns the last value?

We would except that:
 context.result.body = "TypeScript";
    context.result.body = "Rules";
Would not just return Rules

**Comment by ejfasting** [comment]
Hello @Nawksen, 

I see this could be improved, with a simpler example, but the [HTML-sample](https://github.com/SuperOffice/typescript-for-superoffice/blob/main/Examples/HTML/HTMLPage-example.crmscript) showcases how this is done. 
You 'just' append to the context.result.body:
```typescript
const num = ["foo", "bar"];
num.forEach(function (value) {
    context.result.body += value;
}); 
```
So there is no magic here, you have to append all the values to context.result.body, and you are party correct that only the last value is returned.

Hope this explains it (?)

//Eivind
